scripts/tfserp_main.py

```diff
@@ -6,131 +6,129 @@
 
 import numpy as np
 import pandas as pd
-from tfsenc_load_signal import load_electrode_data
-from tfsenc_main import process_subjects, return_stitch_index, write_output
-from tfsenc_parser import parse_arguments
+from tfsenc_config import parse_arguments, setup_environ, write_config
 from tfsenc_read_datum import read_datum
```

Comment:
nit: imports out of order

Reply:
fyi i tend to use isort and black in that order...because they often collide (of course there's a way to bypass that)